### DISCUSSION TITLE
How to create new docker installation with all of the data from SQLite

### DISCUSSION THREAD

**Discussion by Acvarium** [inventree/InvenTree#5430] [General]
I've been using Invetree for a long time. I had it on a server with SQLite3 as a database. After some changes from provider to the server, it stopped working and I decided to, somehow, migrate it to my PC with all of the data intact.
Is there a simple way to take the folder with original data from the server and do the Docker installation with it?
The final version of Inventree on the server was 0.11.2
On the server was Ubuntu 20.04 (now it is 22.04)
On the PC is Manjaro
And also I need to point out, that I am complete noob to the Docker and will need a detail steps

**Comment by SchrodingersGat** [comment]
> Is there a simple way to take the folder with original data from the server and do the Docker installation with it?

Not if you were previously running sqlite - the docker install uses postgres.

You will need to follow the _migration procedure_: https://docs.inventree.org/en/latest/start/migrate/

1. Export data from sqlite database
2. Install new database via docker / docker compose
3. Import data into docker / postgres database
4. Copy media files across
